cli/js/lib.deno.ns.d.ts

```diff
@@ -1283,8 +1283,11 @@ declare namespace Deno {
    *
    *       Deno.renameSync("old/path", "new/path");
    *
-   * Throws error if attempting to rename to a directory which exists and is not
-   * empty.
+   * On Unix, this operation is like `mv -T` in not following symlinks at
+   * either path.
```

Comment:
On OSX `mv -T` doesn't exist. I'm not sure what it does. I think this sentence is overly specific - suggest removing.

Reply:
On GNU `mv` and `cp`, the `-T` flag means that when `b` is a directory, don't reinterpret `mv a b` as meaning `mv a b/a`. I thought I had verified that this flag was on Mac `mv` and `cp`, but I see now that I didn't. On some versions of Mac/BSD, there's a `-h` flag that has some of the same effect.

I'll change to "On Unix, this operation doesn't follow symlinks at either path." (I don't know what it does with symlinks on Windows, and can't experiment because Deno doesn't yet create symlinks there. When we find out, we can change the docstring.)